### ISSUE TITLE
[Suggestion] "Move Items is not supported with this recipe and this container" when using Stonecutter

### ISSUE THREAD

**Issue by epifeny** [shedaniel/RoughlyEnoughItems#793]
### What happened?

I want to take a stack of _Cobbled Deepslate_ and turn it into _Deepslate Tiles_. That's doable using the stonecutter but using REI I'm getting "_Move Items is not supported with this recipe and this container_" while hovering the + sign under the Stone Cutting tab or RIE.

version: `RoughlyEnoughItems-7.3.428`
Fabric: `0.12.12-1.18.1`

![image](https://user-images.githubusercontent.com/89062105/154974155-918c2193-1ec2-4d84-a5ab-93518ed97e56.png)
![image](https://user-images.githubusercontent.com/89062105/154974169-adcf36a3-4ac9-47ac-86a4-f7772c50bc78.png)

### What mod loaders are you seeing the problem on?

Fabric

### What do you think this bug is of?

- [ ] Visual
- [X] Recipe Lookup
- [ ] Cheat Mode
- [ ] Plugin Integration / JEI Plugin Compatibility
- [ ] Others

**Comment by shedaniel** [comment]
Yeah this is intended. I didn't add support for the stonecutter, perhaps we can do in the future, so I am changing this into a feature request.
